packages/kbn-apm-synthtrace/src/scenarios/logs_only.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { LogDocument, log, generateShortId, generateLongId } from '@kbn/apm-synthtrace-client';
+import { Scenario } from '../cli/scenario';
+import { withClient } from '../lib/utils/with_client';
+import { parseLogsScenarioOpts } from './helpers/logs_scenario_opts_parser';
+import { IndexTemplateName } from '../lib/logs/custom_logsdb_index_templates';
+import { getCluster, getCloudRegion, getCloudProvider } from './helpers/logs_mock_data';
+
+const MESSAGE_LOG_LEVELS = [
+  { message: 'A simple log', level: 'info' },
+  { message: 'Yet another debug log', level: 'debug' },
+  { message: 'Error with certificate: "ca_trusted_fingerprint"', level: 'error' },
+];
+
+const scenario: Scenario<LogDocument> = async (runOptions) => {
```

Comment:
nit: we already have a scenario called simple_logs

Reply:
@kpatticha Good point, thanks! The first time I tried it didn't work but after a restart I saw that it covers everything I added in the `logs_only` so I opened a [PR](https://github.com/elastic/kibana/pull/207716) to remove it :)